packages/monorepo-scripts/src/copy_contract_artifacts.ts

```diff
@@ -0,0 +1,41 @@
+const pkgNames = process.argv.slice(2);
+import * as artifacts from '@0x/contract-artifacts';
+import * as fs from 'fs';
+import * as path from 'path';
+/*
+ * @param dir        the directory name under 0x-monorepo/contracts, e.g. 'exchange', 'extensions'
+ * @param artifact   the artifact name without extension, e.g. DutchAuction, Exchange
+ */
+function shouldSkip(dir: string, artifact: string): boolean {
+    return (dir === 'extensions' && artifact === 'Exchange') || (dir === 'extensions' && artifact === 'WETH9');
```

Comment:
Why do we skip?

Reply:
If there is a good reason, please add it to the source code to help the next reader understand.